confluent_kafka_helpers/exceptions.py

```diff
@@ -7,3 +7,11 @@ class KafkaTransportError(Exception):
     Kafka transport errors:
         - GroupCoordinator response error: Local: Broker transport failure
     """
+
+
+class KafkaError(Exception):
+    pass
+
+
+class KafkaDeliveryError(Exception):
```

Comment:
Shouldn't KafkaDeliveryError be a subclass of KafkaError? Makes exception handling easier as well